titty

```diff
@@ -100,7 +100,7 @@ tbackstart(){
 	STARTTIME="$(date -d "$(date -d "-$2")" +%s)"
 	if tcheck_tracking
 	then
-		printf "%s|%s" "$1" "$STARTTIME" >> "$masterlist"
+		printf "%s|%s" "$1" $STARTTIME >> $masterlist
```

Comment:
masterlist is a file name, and file names could have spaces if the user desired, this would lead to and error if `$masterlist` is not quoted

Reply:
Similarly for all other places where `$masterlist` is present